modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -147,6 +153,9 @@ public void setUp() throws Exception {
 			(ObjectScriptingExecutor)_getAndSetFieldValue(
 				ObjectScriptingExecutor.class, "_objectScriptingExecutor",
 				ObjectActionExecutorConstants.KEY_GROOVY);
+		_userObjectDefinition =
+			_objectDefinitionLocalService.fetchObjectDefinitionByClassName(
+				TestPropsValues.getCompanyId(), User.class.getName());
 		_user = UserTestUtil.addUser();
```

Comment:
@murilostodolni to keep it in alphabetical order, `_userObjectDefinition` must come after `_user`